Installation.config

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
1) can we move this file inside Scratch? Maybe into a folder called "templates"? AFAIK this should be an example of an installation.config file. Or we can even get rid of it and just document, that it's an option to have one?
2) if we keep it, remove the data and/or replace it with super generic defaults and remove all nodes not used in the installation.config context.

Reply:
I would suggest to remove it alltogether and document the required format on the readme of this project. What do you think @retohugi @ernstjoss ?